deployer/docker-compose.yml

```
@@ -1,22 +1,21 @@
-version: '3'
-
 services:
   deployer-ms:
-    image: aamdigital/deployer-ms:latest
-    networks:
-      - nginx-proxy_default
+    image: ghcr.io/aam-digital/deployer-backend/deployer-api:pr-8
```

```suggestion
    image: ghcr.io/aam-digital/deployer-backend/deployer-api:pr-8
```

let's merge this and set a proper version before merging this here in ndb-setup